example/javascriptlib/basic/4-multi-modules/library/src/library.ts

```diff
@@ -0,0 +1,70 @@
+import * as fs from 'fs';
```

Comment:
This example code is probably too complicated for its purpose, which is just a stub to show off the build configuration. See `example/javalib/basic/3-multi-module` as an example for roughly how complex it should be (i.e. not very)